config/samples/network_v1alpha1_workspacenetworkpolicy.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: network.kubesphere.io/v1alpha1
+kind: WorkspaceNetworkPolicy
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: workspacenetworkpolicy-sample
+spec:
+  # Add fields here
+  foo: bar
```

Comment:
Is it wrong to input `foo` as key?

Reply:
it is a generated file. would move it out of repo